dockstore-webservice/src/main/resources/openapi.yaml

```diff
@@ -1,6 +1,6 @@
-openapi: 3.0.0
+<html><body><pre>openapi: 3.0.0
```

Comment:
no longer valid when pasted into swagger editor.  any ideas why this is here?

Reply:
I saw this occur with swagger2open.sh and thought I fixed it.
I think it might be a cross ubuntu/Mac bug (as in in the original introduced this on Ubuntu, the fixed version introduces this on Mac?)

Reply:
Yes, that must be because I'm running on a Mac. May need to tackle #1529 first.